.github/workflows/deploy-beta.yml

```diff
@@ -73,12 +73,13 @@ jobs:
           REMOTE_HOST: ${{ secrets.REMOTE_HOST }}
           REMOTE_PATH: beta.destinyitemmanager.com
 
-      - name: Purge CloudFlare cache
-        run: ./build/purge-cloudflare.sh
+      - name: Cloudflare Purge Cache
+        uses: jakejarvis/cloudflare-purge-action@v0.3.0
         env:
+          CLOUDFLARE_ZONE: ${{ secrets.CLOUDFLARE_ZONE }}
           CLOUDFLARE_KEY: ${{ secrets.CLOUDFLARE_KEY }}
           CLOUDFLARE_EMAIL: ${{ secrets.CLOUDFLARE_EMAIL }}
-          APP_DOMAIN: beta.destinyitemmanager.com
+          PURGE_URLS: '["https://beta.destinyitemmanager.com", "https://beta.destinyitemmanager.com/index.html", "https://beta.destinyitemmanager.com/version.json", "https://beta.destinyitemmanager.com/service-worker.js", "https://beta.destinyitemmanager.com/return.html", "https://beta.destinyitemmanager.com/.well-known/assetlinks.json", "https://beta.destinyitemmanager.com/.well-known/apple-app-site-association", "https://beta.destinyitemmanager.com/manifest-webapp-6-2018.json", "https://beta.destinyitemmanager.com/manifest-webapp-6-2018-ios.json"]'
```

Comment:
The thing I liked about our custom script is that we could maintain the list of URLs in one place for both domains.

Reply:
Once I get them all put together I can consolidate them, https://docs.github.com/en/actions/using-workflows/reusing-workflows

Reply:
also adding a flavor environment variable to assist beforehand

Reply:
https://github.com/DestinyItemManager/d2-additional-info/runs/5525233093?check_suite_focus=true

environment variable substitution working